apps/server/src/modules/files-storage/controller/files-storage.controller.ts

```diff
@@ -1,3 +1,4 @@
+import { NotFound } from '@feathersjs/errors';
```

Comment:
Why do you want to use the feathers-exception here instead of the nest-exception?